internal/pkg/webhooks/utils/utils.go

```diff
@@ -45,6 +45,16 @@ func RemoveIfExists(list []string, item string) []string {
 	return list
 }
 
+// Exists checks an item is present in the provided list.
+func Exists(list []string, item string) bool {
```

Comment:
We already have this function in https://github.com/kubernetes-sigs/security-profiles-operator/blob/dd72d74e37a41afa7084d9bf680a3b9dcc95666e/internal/pkg/util/names.go#L36-L44

Reply:
I removed the duplicated function and used the one you are mentioning here. Thanks for pointing this out.